app/main/authorize/access_token_sign_in_required.py

```diff
@@ -57,3 +59,26 @@ def decorated_view(*args, **kwargs):
     decorated_view.access_token_sign_in_required = True
 
     return decorated_view
+
+
+def _validate_access_token():
+    is_valid_access_token = False
+
+    keycloak_openid = keycloak.KeycloakOpenID(
```

Comment:
take a look at generalising a global object. not 100% sure if I want to but will look into .

Reply:
done